### PR TITLE
feat(ui): add push notification on pipeline status

### PR DESCRIPTION
**Pull request by bnjjj**
Display a push notification in browser when your build status change to 'Failing' or 'Success'

### CODE REVIEW

File: ui/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -177,6 +177,8 @@
   "notifications_no" : "Il n'y a aucune notification définie",
   "notification_on_pipeline_s" : "Sur les pipelines : ",
   "notification_on_pipeline" : "Sur le pipeline : ",
+  "notification_on_pipeline_success" : "Votre pipeline '{{pipelineName}}' s'est exécutée avec succès",
```
Comment (sguiheux) [inline ui/src/assets/i18n/fr.json]: éxécuté

File: ui/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -177,6 +177,8 @@
...
+  "notification_on_pipeline_failing" : "Votre pipeline '{{pipelineName}}' est tombée en erreur",
```
Comment (sguiheux) [inline ui/src/assets/i18n/fr.json]: tombé

File: ui/src/app/views/application/show/workflow/tree/item/application.workflow.item.component.ts [hunk 3]
```diff
@@ -288,4 +290,35 @@ export class ApplicationWorkflowItemComponent {
     getTriggerSource(pb: PipelineBuild): string {
         return PipelineBuild.GetTriggerSource(pb);
     }
+
+    handleNotification(pipeline: Pipeline): void {
+        switch (pipeline.last_pipeline_build.status) {
+        case 'Success':
+            this._notification.create(this._translate.instant('notification_on_pipeline_success', {
+                pipelineName: pipeline.name
+            })).subscribe();
```
Comment (sguiheux) [inline ui/src/app/views/application/show/workflow/tree/item/application.workflow.item.component.ts]: no need to destroy subscription on the onDestroy method of the component? ( for this one and all others)

File: ui/src/assets/i18n/fr.json (hunk 1 above)
Comment (fsamin) [inline ui/src/assets/i18n/fr.json]: nope: exécuté

File: ui/src/assets/i18n/en.json [hunk 4]
```diff
@@ -177,6 +177,8 @@
   "notifications_no" : "There is no notification",
   "notification_on_pipeline_s" : "On pipelines: ",
   "notification_on_pipeline" : "On pipeline: ",
+  "notification_on_pipeline_success" : "Your pipeline '{{pipelineName}}' was executed with success",
```
Comment (fsamin) [inline ui/src/assets/i18n/en.json]: has been successfully executed

File: ui/src/assets/i18n/en.json [hunk 5]
```diff
@@ -177,6 +177,8 @@
...
+  "notification_on_pipeline_failing" : "Your pipeline '{{pipelineName}}' was failing",
```
Comment (fsamin) [inline ui/src/assets/i18n/en.json]: has not been successfully executed

**Pull request merged**